docs/spl-docs/evaluation-functions/informational-functions.md

```diff
@@ -9,132 +9,252 @@ This function returns `TRUE` if the given value is Boolean.
 
 ### Usage
 
-You can use this function with the `eval` and `where` commands, within the `WHERE` clause of the `from` command, and as part of evaluation expressions with other commands.
+`isnull` can be used with `eval` and `where` commands to check if a field's value is `NULL`. This is particularly useful in conditional expressions and for filtering records based on the presence of `NULL` values.
 
 ### Example
 
-1. Using the `isnull` function with the `if` function to add a field "n" with values "yes" or "no" depending on whether the "field" is `NULL`:
+1. To mark records with a `NULL` `transactionAmount` as "Not Available":
 
 ```spl
-... | eval n=if(isnull(latency),"yes","no")
+... | eval transactionStatus=if(isnull(transactionAmount), "Not Available", "Available")
 ```
 
-2. Using the `isnull` function with the `where` command to filter results where the `field` is `NULL`:
+2. To exclude records where `transactionAmount` is `NULL`:
 
 ```spl
-... | where isnull(latency)
+... | where isnull(transactionAmount) = FALSE
 ```
 
+### Use-Case Example
+
+**Problem**: A financial transactions dataset includes a `transactionAmount` field. Some transactions are pending and have not been assigned an amount, resulting in `NULL` values. These `NULL` values need to be identified and handled appropriately for accurate financial reporting.
+
+**Solution**: Use the `isnull` function to identify records with `NULL` `transactionAmount` values. This can help in segregating pending transactions from completed ones.
+
+**Implementation**:
+
+```spl
+... | eval transactionStatus=if(isnull(transactionAmount), "Pending", "Completed")
+    | stats count by transactionStatus
+```
+
+**Explanation**:
+
+- The `eval` command uses `isnull` to check for `NULL` `transactionAmount` values, labeling these transactions as "Pending" and others as "Completed".
+- The `stats` command then counts the number of transactions in each category, providing a clear overview of pending versus completed transactions.
+
 ## **isnotnull(&lt;value&gt;)**
 
 This function returns `TRUE` if the given value is not `NULL`.
 
 ### Usage
 
-This function is useful for checking whether a field contains a value. You can use this function with the `eval` and `where` commands, within the `WHERE` clause of the `from` command, and as part of evaluation expressions with other commands.
+`isnotnull` can be used with `eval` and `where` commands to check if a field's value is not `NULL`. This is particularly useful in conditional expressions and for filtering records based on the absence of `NULL` values.
 
 ### Example
 
-1. Using the `isnotnull` function with the `if` function to add a field "n" with values "yes" or "no" depending on whether the "field" is not `NULL`:
+1. To mark records with a non-`NULL` `customerFeedback` as "Received":
 
 ```spl
-... | eval n=if(isnotnull(latency),"yes","no")
+... | eval feedbackStatus=if(isnotnull(customerFeedback), "Received", "Awaiting")
 ```
```

Comment:
Since a log line won't be changed once it's been ingested, I think this example doesn't quite make sense. Because if you first ingested a log line that this query would mark as `Awaiting`, and then the feedback was submitted so you got another log line ingested that this query would mark as `Received`, you'd still have one log line that gets marked as `Awaiting`. Maybe changing `Awaiting` to something else, or using a different example would make this more relevant.

You might be over thinking this. Something like `validate(isnotnull(<some-field>, ...))` could be a good enough example

Similar comment for your `isnull` Use-Case Example